src/guides/v2.3/graphql/functional-testing.md

```diff
@@ -9,7 +9,7 @@ Magento provides API functional tests that can verify extension points in GraphQ
 
 ## Creating a new GraphQL functional test
 
-All GraphQL functional tests should inherit from the generic test case `Magento\TestFramework\TestCase\GraphQlAbstract`. It defines the `graphQlQuery()` method, which should be used to perform Web API calls from tests.
+All GraphQL functional tests should locate in `dev/tests/api-functional/testsuite/Magento/GraphQl/` folder and inherit from the generic test case `Magento\TestFramework\TestCase\GraphQlAbstract`. It defines the `graphQlQuery()` method, which should be used to perform Web API calls from tests.
```

Comment:
```suggestion
All GraphQL functional tests be located in `dev/tests/api-functional/testsuite/Magento/GraphQl/` folder and inherit from the generic test case `Magento\TestFramework\TestCase\GraphQlAbstract`. It defines the `graphQlQuery()` and `graphQlMutation()` methods, which should be used to perform Web API calls from tests.
```

Reply:
**fixed**

Thanks , @rogyar

Reply:
Thank you! Please, take a look at my suggestion above, there's information about `graphQlMutation()` method. I believe we should mention it as well.

Reply:
Fixed.

The `master` branch is locked for now, until 2.3.4 published tomorrow. I'll merge the PR at that time.

Reply:
Thank you, @keharper !

I've squashed commits. So, now PR is ready for merge ;)